### ISSUE TITLE
Bitwarden Integration with DuckDuckGo Android browser

### ISSUE THREAD

**Issue by auriuman78** [bitwarden/android#2845]
### Steps To Reproduce

1. Go to nearly any site that requires sign-in in duckduckgo browser (really doesn't matter much for my situation, I can pick nearly any site and get this behavior) 
2. Not in first tab -- Click on sign in link (or go to sign in fields) 
3. Get bitwarden auto fill prompt, click on it to activate bitwarden
4. Credential bitwarden (bio/pin/pass: agnostic)

### Expected Result

Bitwarden should read the site I'm trying to sign in to and auto select the correct matching credentials to choose from (multiple when applicable). 

### Actual Result

Bitwarden will return only the result for the domain that is open only in my first open tab, regardless of what tab I am actually in. 

Since my first tab is always for an informational site with no login, I get a blank page. 

### Screenshots or Videos

If I can obtain a screen grab of this happening I will attach it after submission. 

### Additional Context

Workaround is available:

I can still search just fine and look up the correct credentials to copy paste into the tab sign-in. This is honestly a first world automation issue. Ten or twelve years ago that was the de facto method for input on nearly all password managers. 

In other words not a critical or even urgent issue. This is what I would call "quality". Definitely not security or operational (though I guess it's kinda operational, just not functional operations).

### Operating System

Android

### Operating System Version

Android 13 (kernel 5.10.136-android12-9-27100597)

### Device

Samsung galaxy 22+ (oneUI v5.1)

### Build Version

TP1A.220624.014.S906USQS3CWI1

### Beta

- [ ] Using a pre-release version of the application.

**Comment by sammbw** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by chak** [comment]
Curious if this will be fixed any time soon?

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/2845
